### DISCUSSION TITLE
Obsidian UI 技术原理

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#23] [General]
# Obsidian UI 技术原理

Obsidian 有比较复杂的 UI 设计，为了能够更好的进行个性化定制和插件开发，UI 元素相关的技术细节值得深入研究，最后附上基于 UI 操作的 local graph 快捷工具的演示

https://www.edony.ink/details-of-obsidian-ui/

**Comment by edonyzpc** [comment]
QuickAdd Macro 定制的 Local Graph 工具后面打算整一下，以 Obsidian Vault 模版的形式提供出来。后续可以关注 repo：[edony-shadow-walker/pkm-template: My PKM system template based on Obsidian](https://github.com/edony-shadow-walker/pkm-template)
